### DISCUSSION TITLE
作为一个代码分享平台，是否应该管理这些与代码分享甚至与代码都不相关的东西？

### DISCUSSION THREAD

**Discussion by Hmis556** [community/community#108422] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

在简中圈中全是关于政治的话题，几乎用任何言语在GitHub上都会出现[https://github.com/cirosantilli/china-dictatorship](url)这个网页，或者关于这个用户的东西[https://github.com/cirosantilli](url) ，你们不应该管管这些滥用公共资源的人，或者推出一个屏蔽装置。

**Comment by queenofcorgis** [comment]
Hi @Hmis556 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.


However, we really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
